### DISCUSSION TITLE
Dramatic editor experience

### DISCUSSION THREAD

**Discussion by xtrasmal** [zed-industries/zed#15192] [Feature Requests]
I live inside my editor so I like to add as much drama as in real life.

Anybody have any tips on how I could get started experimenting, creating artistic and dramatic experiences in editor.rs?

https://github.com/user-attachments/assets/fc811083-2b80-4568-b59c-349a088aea18

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
